bbb/services.py

```diff
@@ -139,12 +122,6 @@ def handleFinished(self, data, msg):
             msg.ack()
             return
 
-        buildername = data["payload"]["build"]["builderName"]
-
-        if not allow_builder(buildername, self.allowed_builders):
-            msg.ack()
-            return
```

Comment:
yay, it's much better to report `malformed-payload`... as oppose to just letting the task sit as pending until deadline :)